RELEASING/set_release_env.sh

```diff
@@ -16,16 +16,27 @@
 # limitations under the License.
 #
 usage() {
-   echo "usage: . set_release_env.sh <SUPERSET_VERSION> <SUPERSET_RC> <PGP_KEY_FULLBANE>"
+   echo "usage: . set_release_env.sh <SUPERSET_RC_VERSION> <PGP_KEY_FULLBANE>"
+   echo "example: . set_relese_env.sh 0.35.2rc1 myid@apache.org"
 }
 
-if [ -z "$1" ] || [ -z "$2" ] || [ -z "$3" ]; then
+if [ -z "$1" ] || [ -z "$2" ]; then
   usage;
 else
-  export SUPERSET_VERSION="${1}"
-  export SUPERSET_RC="${2}"
-  export SUPERSET_PGP_FULLNAME="${3}"
-  export SUPERSET_VERSION_RC="${SUPERSET_VERSION}rc${SUPERSET_RC}"
+  if [[ ${1} =~ ^([0-9]+)\.([0-9]+)\.([0-9]+)rc([0-9]+)$ ]]; then
+    VERSION_MAJOR="${BASH_REMATCH[1]}"
+    VERSION_MINOR="${BASH_REMATCH[2]}"
+    VERSION_PATCH="${BASH_REMATCH[3]}"
+    VERSION_RC="${BASH_REMATCH[4]}"
```

Comment:
I felt it more natural to specify the fully qualified RC version and regex that for major/minor/patch/rc.

Reply:
nice!